src/style.css

```diff
@@ -0,0 +1,39 @@
+* {
+  box-sizing: border-box;
+}
+
+body,
+div {
+  margin: 0;
+  padding: 0;
+}
+
+.stars {
+  display: flex;
+  padding-top: 2px;
+  padding-left: 2px;
+}
+
+.stars:first-child {
+  padding-top: 2px;
+  padding-left: 2px;
+}
+
+.stars__star {
+  background-image: url(./images/star.svg);
+  background-repeat: no-repeat;
+  width: 15px;
+  height: 14px;
```

Comment:
You need to check sizes of stars from design

Reply:
imensions correspond to automatic tests and, accordingly, to the design